monitoring/README.md

```diff
@@ -46,5 +46,8 @@ $ sudo docker run --net=host livepeer/monitoring:latest --mode standalone --node
 
 ```
 
+**note** on OSX omit ``--net=host`  and use `docker.host.internal` instead of localhost for the `--nodes`.
```

Comment:
I had to use `host.docker.internal` to get this to work (also noted by the [docs](https://docs.docker.com/docker-for-mac/networking/)).

Reply:
Oh sorry I turned this into docker.host.internal instead , my bad

Reply:
fixed in c15c362